ios/appservice/appservice.go

```diff
@@ -48,6 +50,27 @@ type AppLaunch struct {
 	Pid int
 }
 
+// AppWithStdIo is the launched app with a connection to the stdio-socket
+type AppWithStdIo struct {
+	stdIoConnection openstdio.Connection
+	AppLaunch
```

Comment:
Now that we this, are there any remaining reasons to have support for app launch without stdio? Maybe that would help us only have one type to represent launches instead of this embedding.

Reply:
I'll give the way it's represented here another look, but the reason why I didn't simply replace it was that if you start the app with stdio you also have to close this connection when you're done. And that means "fire-and-forget" app launches wouldn't be so nice